src/uri/builder.rs

```diff
@@ -78,6 +78,14 @@ impl Builder {
         })
     }
 
+    /// Set the port number for URI, will be part of `Authority`
+    pub fn port(self, port: u16) -> Self {
+        self.map(move |mut parts| {
```

Comment:
I'd move the conversion to here, instead of adding a new field on `Parts`.

Reply:
Done